test/test_find_uvar.py

```diff
@@ -114,15 +120,16 @@ def test_two_triangles_without_faces():
     grid = two_triangles_with_depths()
     grid.faces = None
 
-    grid.save_as_netcdf('2_triangles_without_faces.nc')
-
-    # Read it back in and check it out.
-    ug = UGrid.from_ncfile('2_triangles_without_faces.nc', load_data=True)
+    fname = '2_triangles_without_faces.nc'
+    with chdir(test_files):
+        grid.save_as_netcdf(fname)
+        ug = UGrid.from_ncfile(fname, load_data=True)
+        os.remove(fname)
```

Comment:
Let me think about this in another PR.  In fact, if such feature is desired, we can add as a py.test option.  Because we will want to do that locally and not in the CI.